R/dbi-connection.R

```diff
@@ -148,7 +148,7 @@ setMethod("dbQuoteIdentifier", c("OdbcConnection", "character"),
       return(DBI::SQL(character()))
     }
     if (any(is.na(x))) {
-      stop("Cannot pass NA to dbQuoteIdentifier()", call. = FALSE)
+      cli::cli_abort("NA values are not supported in {.arg x}.")
```

Comment:
According to https://style.tidyverse.org/error-messages.html, we should use "can't", e.g. `{.arg x} can't be NA`